code/processes/orchestrator.q

```diff
@@ -0,0 +1,56 @@
+/TorQ Orchestrator Process
+
+\d .orch
+
+/default parameters
+
+scalingdetails:([] time:`timestamp$(); procname:`$(); dir:`$(); instancecreated:`$(); instanceremoved:`$(); totalnumofinstances:`int$(); lowerlimit:`int$(); upperlimit:`int$());	/table for tracking scaling
+
+processlimitscsv:hsym first .proc.getconfigfile"processlimits.csv";	/location of csv file
+limits:1!("SII";enlist ",")0:processlimitscsv;	/table of scalable processes and the max number of instances allowed for each
```

Comment:
What is the key of the file?  1! doesn't give an indication, I think I'd prefer use of xkey here given we don't know the file.

2 other points - is there already csv loading functionality in TorQ? Could it be leveraged? For example what happens if file doesn't exist here? OR if the file load fails?  We need to handle those scenarios.

Reply:
Ok cool I will have a look for this functionality